docs/user/trusted-publishers/security-model.md

```diff
@@ -4,23 +4,49 @@ title: Security Model and Considerations
 
 # Security model and considerations
 
-## General considerations
-
-While more secure than passwords and long-lived API tokens, OIDC publishing
-is not a panacea. In particular:
-
-* Short-lived API tokens are still sensitive material, and should not be
-  disclosed (ideally not at all, but certainly not before they expire).
-
-* OIDC tokens themselves are sensitive material, and should not be disclosed.
-  OIDC tokens are also short-lived, but an attacker who successfully intercepts
-  one can mint API tokens against it for as long as it lives.
+Trusted Publishing is primarily designed to be a more secure alternative to
+the long-lived API tokens that have traditionally been used for publishing to
+PyPI.
+
+In recent years, theft of credentials such as API tokens has [played a major
+role in cyber attacks]. The reason for this is the unfortunate reality that
+managing credentials can be complicated and risky. Trusted Publishing reduces
+this risk by using short-lived tokens instead of long-lived tokens. Short-lived
+tokens are less likely to be misplaced, leaked in logs, or stolen by malware.
+If short-lived tokens are leaked, they only give attackers a narrow time window
+to exploit the leaked token, which minimizes the potential damage.
```

Comment:
```suggestion
tokens are less likely to be misplaced, leaked in logs, or stolen by malware
since they don't have to be stored. Additionally, if short-lived tokens are
leaked, they only give attackers a narrow time window to exploit the leaked
token, which minimizes the potential damage.
```